### PR TITLE
feat: Add stats widget

### PR DESCRIPTION
**Pull request by pixelastic**
First small contribution to the instantsearch.js :)

image

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
   InstantSearch: require('./lib/InstantSearch'),
   widgets: {
-    searchBox: require('./widgets/search-box/'),
     hits: require('./widgets/hits/'),
+    multipleChoiceList: require('./widgets/multiple-choice-list/'),
     pagination: require('./widgets/pagination/'),
-    multipleChoiceList: require('./widgets/multiple-choice-list/')
+    searchBox: require('./widgets/search-box/'),
+    stats: require('./widgets/stats/')
   }
```
Comment (pixelastic) [inline index.js]: I just re-ordered them alphabetically, to keep it consistent with directory display

File: widgets/pagination/index.js [hunk 2]
```diff
@@ -25,4 +25,4 @@ function hits({container, cssClass, labels, maxPages} = {}) {
   };
 }
 
-module.exports = hits;
+module.exports = pagination;
```
Comment (pixelastic) [inline widgets/pagination/index.js]: Fixed a bad copy-paste while I was here ;)

File: widgets/stats/template.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<div>
+  {{nbHits}} results
```
Comment (redox) [inline widgets/stats/template.html]: Can you also handle the singular form for `results` when 1 result?

**Comment by pixelastic** [comment]
I've passed an `isPlural` to handle this kind of logic in the template (Mustache can't check if a variable is of a specific value, it can only check truthy/falsy)

**Comment by pixelastic** [comment]
Hmm ok, so `hasZeroResults`, `hasOneResult` and `hasManyResults`. Will populate those fields with the english logic at first and we'll handle languages that have different definition of "many" (3+, 4+) later on.

File: index.js (hunk 1 above)
Comment (vvo) [inline index.js]: good

**Comment by vvo** [comment]
Perfect!

**Pull request merged**